battery-status/battery-charging-manual.html

```diff
@@ -31,10 +31,10 @@
   navigator.getBattery().then(function (battery) {
     t.step(function () {
       assert_true(battery.charging, 'charging must be set to true');
-      assert_less_than(battery.chargingTime, Infinity, 'chargingTime must be less than Infinity');
+      assert_less_than_equal(battery.chargingTime, Infinity, 'chargingTime must be less than or equal to Infinity');
```

Comment:
While I agree this is correct, I'm not sure there is much point to that assert — `x ≤ ∞` is always true.